### ISSUE TITLE
Make the root document smaller

### ISSUE THREAD

**Issue by wjdavis5** [Azure/Azure-Functions#1543]
The root document when you create a new function:

![image](https://user-images.githubusercontent.com/2580575/78067643-12a64880-7365-11ea-9b1d-18f707c7d612.png)

Is pretty large considering what it does.

![image](https://user-images.githubusercontent.com/2580575/78067672-1e920a80-7365-11ea-8e04-a646466646ec.png)

Why does that page need jquery? This should be a tiny static site.
Why does it matter? Consider this:

I create a new function app, and lets say this function is in the east and in the west.
I slap an instance of FrontDoor in front of it, enabling waf, or any other features and I leave the healthcheck as '/'

Now Front door is hitting the root of the azure functions from 90 locations globally every minute, or whatever the default is. Serving up 450.95 KB of content each time.

I also have App Insights hooked up, and wouldnt ya know it, now I'm generating over $200 / mnth in app insights logging serving up this page.

I'm sure there are probably going to be comments about why would you leave the healthcheck at '/'. But really the question "Why are you using jquery on this page?" is still valid.

**Comment by anthonychu** [comment]
You can modify the Front Door probe to a HEAD request.

You can also disable the page by setting the `AzureWebJobsDisableHomepage` app setting to `true`, although the page will then respond with a 204 status instead of 200. Not sure if Front Door see that has healthy.

I'm unable to see any traces or requests in App Insights relating to this page. If you have any logs that you see, please share so we can provide some guidance on suppressing them.

**Comment by wjdavis5** [comment]
If you read my original post on this thread I give the detail you're missing. The problem is when you create a new FD instance and leave the default Healthcheck path of '/'. When you do this and you have app insights enabled you'll see lots of requests from the healthcheck probe hitting the root document.

**Comment by anthonychu** [comment]
Understood. I’m not able to reproduce the app insights logs on the calls to the homepage. Need an example to better understand the problem.

**Comment by wjdavis5** [comment]
Ok let me see if I can help clarify.

- Create 1 azure function on consumption plan - dont really even need to deploy code to it.
- Create Azure FD (not standard/premium preview).
- Configure Azure FD. Health Check route should be '/' and request is GET. (I did notice that its defaulting to HEAD now, that was not always the case).
- I have app insights connected and log analytics integrated

After creating this I am no longer seeing the change in daily costs associated with Log Analytics that I originally opened this about. We did have cases opened on this some time ago but it does appear to have been fixed.

That said I am now seeing a bulk of the costs is associated with Bandwidth. So an empty Azure Function is generating ~7/day in costs for bandwidth fully attributed to health check requests to '/'. 
I still think its valid to say the default document doesnt need to be using jquery and a static html page would be more than sufficient and would decrease these costs.

Either way I'm going to close this out, I think you understand my point.
